roles/roce_backend/vars/main.yml

```diff
@@ -0,0 +1,45 @@
+---
+# vars file for roce_backend
+# Physical function adapter names which part of RoCE backend
+sriov_resources:
+  - pf_name: enp5s0f0
+    vlan_id: 111
+    res_name: "sriov_111"
+    network_name: sriov111
+  - pf_name: enp11s0f0
+    vlan_id: 112
+    res_name: "sriov_112"
+    network_name: sriov112
+  - pf_name: ens11f0
+    vlan_id: 113
+    res_name: "sriov_113"
+    network_name: sriov113
+  - pf_name: ens12f0
+    vlan_id: 114
+    res_name: "sriov_114"
+    network_name: sriov114
+
+# Virtual function device ID. Default - "MT28908 Family [ConnectX-6 Virtual Function]"
+# Detailed information about all Mellanox Device ID can be found - https://devicehunt.com/view/type/pci/vendor/15B3.
+# Supported values 
+#    101c - MT28908 Family [ConnectX-6 Virtual Function]
+#    101a - MT28800 Family [ConnectX-5 Ex Virtual Function]
+#    1018 - MT27800 Family [ConnectX-5 Virtual Function]   
+#    1016 - MT27710 Family [ConnectX-4 Lx Virtual Function]
+#    1014 - MT27700 Family [ConnectX-4 Virtual Function]
+vendor: "15b3"
+dev_id: "101c"
```

Comment:
Our `dev_id` was 1018. We had to be clever about how it was quoted otherwise later on during configmap setup this variable substitution was interpreted as a number instead of text and we got the following error:
```
$ kubectl logs  -n kube-system    kube-sriov-device-plugin-amd64-9gbdp
I0129 18:44:53.708730   27636 manager.go:70] Using Kubelet Plugin Registry Mode
I0129 18:44:53.709070   27636 main.go:44] resource manager reading configs
E0129 18:44:53.709218   27636 main.go:46] error getting resources from file error unmarshalling raw bytes json: cannot unmarshal number into Go struct field .resourceList.selectors.devices of type string
```

The fix:
```
dev_id: "'1018'"
```
Note the double quotes around single quotes fixed it. Maybe make a comment or point this out or perhaps fix it by quoting in file: `config_dp.j2`